app/wp-content/themes/kelly/style.css

```diff
@@ -583,11 +583,11 @@ a:active {
 	background-position: center center;
 	background-size: cover;
 }
-.has-post-thumbnail .entry-title .title-overlay {
+.has-post-thumbnail .entry-title .title-overlay, .entry-title .title-overlay {
```

Comment:
Why not just remove `.has-post-thumbnail` from these two lines and leave them as `.entry-title, .title-overlay`?